apm-sniffer/apm-agent/pom.xml

```diff
@@ -151,7 +151,7 @@
                                     <exclude>*:gson</exclude>
                                     <exclude>io.grpc:*</exclude>
                                     <exclude>io.netty:*</exclude>
-                                    <exclude>com.google.protobuf:*</exclude>
+                                    <exclude>com.google.*:*</exclude>
```

Comment:
Exclude all jars that the prefix name is `com.google`. because it can avoid missing some packages